rawfile/utils/logs.py

```diff
@@ -1,39 +1,101 @@
 import time
 import functools
+import sys
+import json
+from loguru import logger
+from loguru._defaults import LOGURU_FORMAT
+from enum import StrEnum
+from google.protobuf.json_format import MessageToDict
 
 
-def indent(obj, lvl):
-    return "\n".join([(lvl * " ") + line for line in str(obj).splitlines()])
+class LoggingFormats(StrEnum):
+    JSON = "json"
+    HUMAN = "human"
```

Comment:
should this be `pretty` ?
```suggestion
    PRETTY = "pretty"
```

Reply:
I'm not good at selecting names :/

Reply:
human was good, only because `pretty` format is used in logging